### ISSUE TITLE
PresenceLight hangs on authentication to Teams. 

### ISSUE THREAD

**Issue by davpeet** [isaacrlevin/PresenceLight#146]
**Describe the bug**
If authentication fails once then when the application is restarted it either hangs while signing in or the application fails to start. 

**To Reproduce**
Start the application.
On the sign in tab click sign in
In the Web Browser which opens to authenticate to Teams close the browser tab without authenticating. 
PresenceLight hangs waiting on authentication (no time out).

**Expected behavior**
Authentication should time out and allow the user to retry authentication.

**Screenshots**
N/A

## Further details
Windows 10

**Comment by isaacrlevin** [comment]
Try to uninstall and reinstall the app

**Comment by isaacrlevin** [comment]
Duplicate of #145
